tests/IndexTest.cpp

```diff
@@ -50,7 +51,7 @@ TEST_CASE("indexes files", "[Index]") {
                                testFile, testFile + ".zindex");
         unique_ptr<LineIndexer> indexer(new RegExpIndexer("^Line ([0-9]+)"));
         builder
-                .addIndexer("default", "blah", true, true, false, move(indexer))
+                .addIndexer("default", "blah", Index::IndexConfig{ true, true, false, false }, move(indexer))
```

Comment:
if you make the changes from above I'd suggest making some named configs to use in these tests. "Index::IndexConfig uniqueNumeric;" etc

It'd be overkill but I often use the "builder" pattern for this:

`Index::IndexConfig().wUnique(true).wNumeric(true)` or similar. Anything's good though!